From c6c7c61f00442d919e2d7179019ce720a248c468 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Mon, 30 Mar 2020 17:29:57 -0700
Subject: [PATCH 1300/1852] staging: apf: Make the APF config as a boolean
 state

commit 8df5cea3b28f4bad9695417289e65acbbbfbcd25 from
https://github.com/Xilinx/linux-xlnx.git

The xlnx.c uses low level functions such as __dma_map_area, that are
not exposed symbols. So it can't build as a separate kernel module,
but should be built as a kernel. So make the config as a boolean.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reported-by: Florian Klink <flokli@flokli.de>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/staging/apf/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/apf/Kconfig b/drivers/staging/apf/Kconfig
index 40d11d02e92d..33cf32d43d02 100644
--- a/drivers/staging/apf/Kconfig
+++ b/drivers/staging/apf/Kconfig
@@ -3,7 +3,7 @@
 #
 
 menuconfig XILINX_APF
-	tristate "Xilinx APF Accelerator driver"
+	bool "Xilinx APF Accelerator driver"
 	depends on ARCH_ZYNQ || ARCH_ZYNQMP
 	default n
 	select UIO
-- 
2.31.1

